lib/audit_log/audit_log/formatter.rb

```diff
@@ -56,12 +56,19 @@ def sidekiq_current_ctx
               Sidekiq::Context.current
             else
               # versions up to 6.0.0
-              Thread.current[:sidekiq_context]
+              parse_sidekiq_ctx(Thread.current[:sidekiq_context])
             end
-          rescue NoMethodError
+          rescue StandardError
             nil
           end
 
+          def parse_sidekiq_ctx(ctx)
+            return unless ctx
+
+            curr_ctx = ctx[-1]
```

Comment:
Why `-1`?

Reply:
It seems that sidekiq appends every context, and we want to use the latest/effective one. Check https://github.com/mperham/sidekiq/blob/v5.2.7/lib/sidekiq/logging.rb#L47

`.last` might be used instead...

Reply:
Yeah, `.last` is more readable

Reply:
Updated to `last`.